src/System.Management.Automation/engine/remoting/fanin/InitialSessionStateProvider.cs

```diff
@@ -52,9 +52,7 @@ internal class ConfigurationDataFromXML
         internal const string MAXPSVERSIONTOKEN = "MaxPSVersion";
         internal const string MODULESTOIMPORT = "ModulesToImport";
         internal const string HOSTMODE = "hostmode";
-        internal const string ENDPOINTCONFIGURATIONTYPE = "sessiontype";
         internal const string WORKFLOWCOREASSEMBLY = "Microsoft.PowerShell.Workflow.ServiceCore, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL";
```

Comment:
We can remove this (WORKFLOWCOREASSEMBLY) and all code accessing it.

Reply:
Will do in follow PRs.

Reply:
@iSazonov Thanks for doing this!  It is great to get this cleaned up.